test/quantization/core/test_workflow_module.py

```diff
@@ -899,6 +900,57 @@ def forward(self, x):
         input = torch.randn(4, 1, 4, 4, device=device)
         model(input)
 
+    @unittest.skipIf(not TEST_MULTIGPU, "multi-GPU not supported")
+    @unittest.skipIf(not TEST_CUDA, "CUDA unavailable")
+    def test_model_parallel(self):
```

Comment:
thanks for creating the test case! To confirm, does this test case fail before the fix and pass after the fix?